docs/guides/getting-started-java-spring.md

```diff
@@ -113,7 +114,7 @@ To implement a service task, take the following steps:
 
 ### Configure Spring Boot Starter
 
-Add the following Maven dependency to your Spring Boot Starter project, replacing `x` with the latest patch level available:
+Add the following Maven dependency to your Spring Boot Starter project's pom.xml file, replacing `x` with [the latest patch level available](https://mvnrepository.com/artifact/io.camunda/spring-boot-starter-camunda-sdk):
```

Comment:
1. `pom.xml`: I had a moment of panic as I forgot where the dependencies go in a spring project 😭.
2. I personally can't ever figure out what the latest patch level is, I wouldn't expect a reader to know either. Linking them to maven central seems like a quick way to figure that out.

Reply:
Ideally, we have some automated way to increment this instead of "x" but I like your addition of a link here for now.